features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -1358,15 +1363,30 @@
             <Permissions>/permission/admin/manage/identity/userstore/config/view</Permissions>
             <Scopes>internal_userstore_view</Scopes>
         </Resource>
-        <Resource context="(.*)/api/server/v1/me/sessions(.*)" secured="true"
+        <Resource context="(.*)/api/users/v1/me/sessions(.*)" secured="true"
                   http-method="all">
             <Permissions>none</Permissions>
             <Scopes>internal_login</Scopes>
         </Resource>
-        <Resource context="(.*)/api/server/v1/(.*)/sessions(.*)" secured="true" http-method="all">
+        <Resource context="(.*)/api/users/v1/(.*)/sessions(.*)" secured="true" http-method="GET">
             <Permissions>/permission/admin/manage/identity/authentication/session/view</Permissions>
             <Scopes>internal_session_view</Scopes>
         </Resource>
+        <Resource context="(.*)/api/users/v1/(.*)/sessions(.*)" secured="true" http-method="DELETE">
+            <Permissions>/permission/admin/manage/identity/authentication/session/delete</Permissions>
+            <Scopes>internal_session_delete</Scopes>
+        </Resource>
+        <Resource context="(.*)/api/users/v1/me(.*)" secured="true" http-method="GET, HEAD, POST, PUT, DELETE, PATCH">
+            <Permissions>none</Permissions>
+            <Scopes>internal_login</Scopes>
+        </Resource>
+        <Resource context="(.*)/api/users/v1/(.*)" secured="true" http-method="all">
+            <Permissions>/permission/admin/manage/identity</Permissions>
+            <Scopes>internal_identity_mgt_view</Scopes>
+            <Scopes>internal_identity_mgt_update</Scopes>
+            <Scopes>internal_identity_mgt_create</Scopes>
+            <Scopes>internal_identity_mgt_delete</Scopes>
+        </Resource>
```

Comment:
Should we break this according to the http-methods and set the relevant scope for each?

Reply:
No, since it is general API we don't need to split it.